src/instructlab/model/train.py

```diff
@@ -560,6 +560,25 @@ def get_files(directory: str, pattern: str) -> list[str]:
         # try to load journal. May be empty.
         journal = TrainingJournal(journalfile=training_journal)
         click.secho("\n\n~~~~~~~~~~~~~STARTING MULTI-PHASE TRAINING~~~~~~~~~~~~~")
+
+        # experimental preference.
+        if phased_phase1_num_epochs != 7:
```

Comment:
Does it necessarily have to be a 7, or can we just take the checkpoint at the end of this phase?

Reply:
We take the last checkpoint, but if the user doesn't specify 7 we let them know that they might want that value.